EasyPost/Models/API/Options.cs

```diff
@@ -262,26 +298,40 @@ public class Options : EasyPostObject
         ///     </list>
         /// </summary>
         [JsonProperty("delivery_confirmation")]
-        public string? DeliveryConfirmation { get; set; }
-        [JsonProperty("delivery_time_preference")]
-        public string? DeliveryTimePreference { get; set; }
+        public string? DeliveryConfirmation { get; set; } // any
+
+        [JsonProperty("delivery_date")]
+        public string? DeliveryDate { get; set; }
 
         /// <summary>
-        ///     The earliest a package should be delivered.
+        ///     The latest a package should be delivered.
         ///     Supported carriers vary.
         /// </summary>
-        [JsonProperty("delivery_min_datetime")]
-        public string? DeliveryMinDatetime { get; set; }
+        [JsonProperty("delivery_max_datetime")]
+        public DateTime? DeliveryMaxDatetime { get; set; } // datetime
 
         /// <summary>
-        ///     The latest a package should be delivered.
+        ///     The earliest a package should be delivered.
         ///     Supported carriers vary.
         /// </summary>
-        [JsonProperty("delivery_max_datetime")]
-        public string? DeliveryMaxDatetime { get; set; }
+        [JsonProperty("delivery_min_datetime")]
+        public DateTime? DeliveryMinDatetime { get; set; } // datetime
```

Comment:
question: are we sure we want datetimes? wasn't there just recently a bug report where this was not acting as a user expected?

Reply:
IIRC the bug was that it was causing problems cuz it was a string instead of a DateTime